Evergreen/Apps/Get-Minitab.ps1

```diff
@@ -0,0 +1,58 @@
+﻿Function Get-Minitab {
+    <#
+        .SYNOPSIS
+            Get the current version and download URI for the supported releases of Minitab.
+
+        .NOTES
+            Author: Andrew Cooper
+            Twitter: @adotcoop
+    #>
+    [CmdletBinding(SupportsShouldProcess = $False)]
+    [OutputType([System.Management.Automation.PSObject])]
+    param (
+        [Parameter(Mandatory = $False, Position = 0)]
+        [ValidateNotNull()]
+        [System.Management.Automation.PSObject]
+        $res = (Get-FunctionResource -AppName ("$($MyInvocation.MyCommand)".Split("-"))[1]),
+
+        [Parameter(Mandatory = $False, Position = 1)]
+        [ValidateNotNull()]
+        [System.String] $Filter
+    )
+
+    ForEach ($Release in $res.Get.Download.Releases) {
+
+        # Build the update uri based on the release number
+        $uri = $res.Get.Update.Uri -replace $res.Get.Update.ReplaceRelease, $Release
+
+        # Query the update feed
+        $params = @{
+            Uri             = $uri
+            UseBasicParsing = $True
+        }
+        $Updatefeed = (Invoke-WebRequest @params).Content
+
+        # Convert from unicode
+        $Updates = [System.Text.Encoding]::Unicode.GetString($Updatefeed)
+        
+        # Get the URI(s) and Version(s) from the Ini file
+        $URIs = [RegEx]::Matches($Updates, $res.Get.Update.MatchFile) | Select-Object -ExpandProperty Value
```

Comment:
Add a `try/catch` block here to handle `$LatestVersion` and `$LatestURI` gracefully in the event that strings aren't matched. Provide feedback in the `catch` with `throw` so that the function provides something if the strings can't be matched. I need to do this more consistently across some other functions too.